### PR TITLE
feat: Added analytics for new features - journal and following

### PR DESCRIPTION
**Pull request by vinnyjth**
### How do I test this PR?

On tempaltes, change your `AnalyticsProvider` to
```
        <AnalyticsProvider trackFunctions={[console.warn]}>
```

Then like, flag, follow, etc.

### CODE REVIEW

File: packages/apollos-ui-connected/src/NodeSingleConnected/index.js [hunk 1]
```diff
@@ -135,6 +136,8 @@ NodeSingleConnectedWithMedia.defaultProps = {
   Component: NodeSingleInner,
 };
 
+export { GET_MEDIA, GET_TITLE };
```
Comment (redreceipt) [inline packages/apollos-ui-connected/src/NodeSingleConnected/index.js:139]: is there a reason you're exporting these here?

**Comment by redreceipt** [comment]
this is awesome

File: packages/apollos-ui-connected/src/NodeSingleConnected/index.js (hunk 1 above)
Comment (vinnyjth) [reply packages/apollos-ui-connected/src/NodeSingleConnected/index.js:139]: I noticed that we use a `GET_TITLE` query in templates, and I figure it's always good to export our queries.

Comment (vinnyjth) [reply packages/apollos-ui-connected/src/NodeSingleConnected/index.js:139]: oh doh, lol

Comment (vinnyjth) [reply packages/apollos-ui-connected/src/NodeSingleConnected/index.js:139]: fixed

**Review by redreceipt — APPROVED** [review]

**Pull request merged**